### DISCUSSION TITLE
Escape and resume with FBW bus

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5873] [General]
Hi dev team and simmers,

ahhrggg... due to a phone call I pressed escape to pause the flight. After the phone call I tried to press resume ..... it is not working :-( ... Asobo issue or FBW or a combination ..... too bad, I was turning into final ..... buhuhuhuhuh ....

![grafik](https://user-images.githubusercontent.com/76531893/134181001-f930293b-2b61-47d6-b3ce-ef4b3c387c8c.png)

greets

**Comment by tracernz** [comment]
The ESC key wouldn't do it?

**Reply by CptBayer** [reply]
Hi tracernz,

no, ESC won´t work .... no chance .... :.p .... 

Greets Klaus

**Reply by tracernz** [reply]
Unfortunately the pause menu seems to have a few issues lately, and also the bug where the mouse pointer disappears. :(
